modules/phase_field/src/action/EulerAngle2RGBAction.C

```diff
@@ -22,6 +22,7 @@ InputParameters validParams<EulerAngle2RGBAction>()
   params.addRequiredParam<MooseEnum>("crystal_structure",structure_enum,"Crystal structure of the material");
   params.addRequiredParam<UserObjectName>("euler_angle_provider", "Name of Euler angle provider user object");
   params.addRequiredParam<UserObjectName>("grain_tracker_object", "The GrainTracker UserObject to get values from.");
+  params.addParam<std::vector<Real> >("no_grain_color", std::vector<Real>{ 0, 0 , 0 }, "RGB value of color used to represent area with no grains");
```

Comment:
Let's not do this at all. I suspect that my newly identified bug will take care of this.